editor/src/messages/tool/common_functionality/measure.rs

```diff
@@ -13,21 +13,37 @@ fn draw_dashed_line(line_start: DVec2, line_end: DVec2, transform: DAffine2, ove
 }
 
 /// Draws a solid line with a length annotation between two points transformed by the given affine transformations.
-fn draw_line_with_length(line_start: DVec2, line_end: DVec2, transform: DAffine2, document_to_viewport: DAffine2, overlay_context: &mut OverlayContext) {
+fn draw_line_with_length(
+	line_start: DVec2,
+	line_end: DVec2,
+	transform: DAffine2,
+	document_to_viewport: DAffine2,
+	overlay_context: &mut OverlayContext,
+	is_vertical_line: Option<bool>,
+	text_on_left: Option<bool>,
+	text_on_top: Option<bool>,
```

Comment:
I don't see anywhere that this function is called with `None` for any of these three. So I believe you can just remove the `Option` wrapper from these three parameters?

Reply:
Sure, that would read better anyway.